services/publicapi/service.go

```diff
@@ -145,6 +156,110 @@ func toTxOutput(transactionOutput *txResponse) *services.SendTransactionOutput {
 	return &services.SendTransactionOutput{ClientResponse: response.Build()}
 }
 
+func (s *service) CallMethod(input *services.CallMethodInput) (*services.CallMethodOutput, error) {
```

Comment:
consider splitting separate methods to separate files

Reply:
will consider when i do call method